### ISSUE TITLE
Volume issue (again)

### ISSUE THREAD

**Issue by iorellana** [rpetrich/libactivator#163]
Hello, im using an ipad mini and when i download the activator, no matter what action i asign to the volumen down, it mutes the device, i mean not mute, just pop all the way down the volume.

**Comment by dchaws** [comment]
I think I am having the same issue. A single press of the volume down button will lower the volume to 0. I reset to default settings. Still happening. iPad retina, iOS 6.1, evasi0n jailbreak. Using activator 1.7.2.

Edit: This bug is not happening on my iPhone 4S iOs 6.1, evasiOn jailbreak, activator 1.7.2.
Great app!

**Comment by iorellana** [comment]
If u unistall the activator via cydia u will get rid of the problem, but we dont want that haha 

Nacho.
http://www.foxdev.com.ar/

```
"Imagination is more important than knowledge. For knowledge is limited, whereas imagination embraces the entire world, stimulating progress, giving birth to evolution. It is, strictly speaking, a real factor in scientific research."

--- Albert Einstein.
```

-----Original Message-----

**Comment by dchaws** [comment]
Issue solved when I updated to Activator 1.7.3.

**Comment by rpetrich** [comment]
Yup, now resolved.
